docs(datetime): adds playground for styling calendar days using shadow parts (#3101)
ionic-team/ionic-docs: docs/api/datetime.md

<GlobalTheming />

### Calendar Days

> The calendar days in the grid-style of an `ion-datetime` can be styled using CSS shadow parts.

Reviewer: ```suggestion
The calendar days in a grid-style `ion-datetime` can be styled using CSS shadow parts.
```
Less awkward wording.
